### ISSUE TITLE
Add Analytics tags on buttons

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#27]

**Comment by nikhilwoodruff** [comment]
Made a start, analytics is up and running but we could add plenty more event loggers

**Comment by anth-volk** [comment]
Are we still doing this?

**Comment by nikhilwoodruff** [comment]
I think it's still worthwhile

**Comment by dotslashbit** [comment]
@anth-volk Can you provide some more clarification to this. Do we need to add analytics tag to each and every event or there's some specific condition on where to put analytics tag

**Comment by anth-volk** [comment]
Let me provide some background first, because perhaps we could use your best judgment on this. We're hoping to track a few things - the number of times users access certain features/functions of the app, how users access these features/functions, and who these users are (where possible). We're not interested so much in tracking particular events as we are particular user flows: if someone clicks on the button to implement a reform on IRS income tax bracket 7, what are all of the next things they do within their user session? How are they going about it? What platform are they on? And what sort of user are they (again, where possible)? 

Perhaps the best first step to this would be developing some sort of system to encapsulate that user flow. Note that we also provide users a cookie consent pop-up, so we can log cookies. 

Do you have any ideas as to how we might proceed with this? Happy to discuss.

**Comment by SEEDART007** [comment]
@anth-volk we can use google analytics and track for the users who clicked it. this is the initial step further we can improve it more. what you think?

**Comment by anth-volk** [comment]
@SEEDART007 I agree. One thing to note - we are in the midst of a redesign, so the highest-value thing we could do here is not so much integrating the code, as it is writing something easily transferable, modular, and clean.

**Comment by SEEDART007** [comment]
@anth-volk ok i see

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
